src/content/blog/2024-02-21-eslint-2023-year-review.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: ESLint's 2023 year in review
+teaser: "2023 saw the much-anticipated release of our new configuration system and more."
+tags:
+  - Updates
+  - Features
+  - Releases
+authors:
+  - nzakas
+categories:
+  - Storytime
+---
+
+The plan for 2023 centered around our first major release [since 2021](https://eslint.org/blog/2021/10/eslint-v8.0.0-released/), ESLint v9.0.0. Most of the year was spent preparing for the release, including the building out of the new configuration system (flat config) and communicating the impact to the ecosystem throughout the year.
```

Comment:
```suggestion
The plan for 2023 centered around our first major release [since 2021](https://eslint.org/blog/2021/10/eslint-v8.0.0-released/), ESLint v9.0.0. Most of the year was spent preparing for the release, building out the new configuration system (flat config) and communicating the impact to the ecosystem throughout the year.
```

this part could be rephrased?